ACLWeb.js

```diff
@@ -1,15 +1,15 @@
 {
 	"translatorID": "f4a5876a-3e53-40e2-9032-d99a30d7a6fc",
 	"label": "ACL",
-	"creator": "Nathan Schneider",
+	"creator": "Nathan Schneider, Yoav Goldberg",
 	"target": "^https?://(www[.])?aclweb\\.org/anthology/[^#]+",
-	"minVersion": "1.0.7",
+	"minVersion": "1.0.8",
```

Comment:
This is the Zotero version that is required for the translator. Should be `3.0` due to `Zotero.selectItems`